build.sh

```diff
@@ -40,34 +62,51 @@ pushd $WEBRTC_SRC
 git checkout $COMMIT
 popd
 
-echo "Generating build scripts ..."
-pushd $WEBRTC_SRC
-python webrtc/build/gyp_webrtc
+echo "Cleaning webrtc ..."
+pushd $WEBRTC_SRC || exit 1
+rm -rf out/$CONFIG
+popd
+
+echo "Applying webrtc patches ..."
+pushd $WEBRTC_SRC || exit 1
+for PATCH in build_at_webrtc_branch_heads_52.patch; do
+	git apply --check ${PROJECT_DIR}/webrtc_patches/${PATCH} || exit 1
+	git am < ${PROJECT_DIR}/webrtc_patches/${PATCH} || exit 1
+done
 popd
 
 echo "Building webrtc ..."
 pushd $WEBRTC_SRC
-ninja -C out/$CONFIG
+export GYP_DEFINES="include_tests=0 include_examples=0"
+python webrtc/build/gyp_webrtc webrtc/api/api.gyp || exit 1
+ninja -C out/$CONFIG || exit 1
 popd
 
 echo "Copying headers ..."
-pushd $WEBRTC_SRC
+pushd $WEBRTC_SRC || exit 1
+rm -rf "$INCLUDE_DIR"
 for h in $(find webrtc/ -type f -name '*.h')
 do
 	mkdir -p "$INCLUDE_DIR/$(dirname $h)"
 	cp $h "$INCLUDE_DIR/$h"
 done
 popd
+pushd $PROJECT_DIR || exit 1
+git clean -fdx "$INCLUDE_DIR"
+popd
 
 echo "Concatenating libraries ..."
 pushd $WEBRTC_SRC/out/$CONFIG
-# on osx:
-# ls *.a > filelist
-# libtool -static -o libwebrtc-magic.a -filelist filelist
-# strip -S -x -o libwebrtc-magic.a libwebrtc-magic.a
-# on linux:
-# ar crs libwebrtc-magic.a $(find . -name '*.o' -not -name '*.main.o')
-mv libwebrtc-magic.a $LIB_DIR/libwebrtc-$OS-$ARCH-magic.a
+if [ "$OS" = "darwin" ]; then
+	ls *.a > filelist
+	libtool -static -o libwebrtc-magic.a -filelist filelist
+	strip -S -x -o libwebrtc-magic.a libwebrtc-magic.a
+else
+	ar crs libwebrtc-magic.a $(find . -name '*.o' -not -name '*.main.o' -not -name 'dump_syms_regtest.o')
```

Comment:
Oops, that was left in from a previous attempt. It is not needed in the current way of building